core/src/main/java/io/ultrabrew/metrics/reporters/SLF4JReporter.java

```diff
@@ -54,7 +63,8 @@ public SLF4JReporter(final String name) {
    * @param windowSizeSeconds window size in seconds
    */
   public SLF4JReporter(final String name, final int windowSizeSeconds) {
-    this(name, " ", " ", " ", windowSizeSeconds);
+    this(name, DEFAULT_TAG_DELIMITER, DEFAULT_FIELD_DELIMITER, DEFAULT_TAGFIELD_DELIMITER,
```

Comment:
I really dislike this as now I have to scroll somewhere and find out what the hell are the values here.

Reply:
I extracted this constants to make it configurable through the Builder. If we don't want to make them configurable then I can inline them.

Reply:
? I am confused, how does replacing `static final DEFAULT_TAG_DELIMITER = " "` with inline `" "` change anything except readability?

Reply:
I pull out a constant to use it in the builder. If you really hate this then I will have to inline them in two places.